packages/access-client/src/stores/store-indexeddb.js

```diff
@@ -149,25 +150,26 @@ export class StoreIndexedDB {
         /** @type {import('p-defer').DeferredPromise<Store>} */
         const { resolve, reject, promise } = defer()
 
-        const dels = []
-
-        for (const [key, value] of data.delegations) {
-          dels.push([
-            key,
-            {
-              meta: value.meta,
-              delegation: [...value.delegation.export()],
-            },
-          ])
-        }
-        const putReq = store.put({
+        /** @type {import('./types').StoreDataIDB} */
+        const raw = {
           id: DATA_ID,
           meta: data.meta,
+          // @ts-ignore
```

Comment:
expect so we get the nudge to remove this if the types align in the future?
```suggestion
          // @ts-expect-error
```